packages/front-end/components/Experiment/NewExperimentForm.tsx

```diff
@@ -472,6 +502,51 @@ const NewExperimentForm: FC<NewExperimentFormProps> = ({
                 datasource project is deleted.
               </div>
             )}
+            {availableTemplates.length >= 1 && !isBandit && (
```

Comment:
I don't think templates should be here for "Import existing experiment", since they can lead to some weird conflicts.

Reply:
Same for "Duplicate Experiment"